### PR TITLE
Movie detailss

### PR DESCRIPTION
**Pull request by Marianna-Weychan**
I created an APIBox in which there are API sources divided into individual components.
Please check the correctness of the code in terms of logic as well as in terms of your components

### CODE REVIEW

File: src/core/APIBox/apiSource.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const apiKey = "d3f19b5007aaab7cb579f83b9a664dec";
+export const apiBaseUrl = "https://www.themoviedb.org/";
```
Comment (AdrianJar) [inline src/core/APIBox/apiSource.js]: here it should be this -> "https://api.themoviedb.org/3" you forgot to type 3 after .org/

**Review by AdrianJar — CHANGES_REQUESTED** [review]
Nice job this will help us all to speed up work

**Review by KatarzynaZaloba — APPROVED** [review]
Can we test it out?

**Review by AdrianJar — APPROVED** [review]

File: src/core/APIBox/apiSource.js (hunk 1 above)
Comment (ghost) [reply src/core/APIBox/apiSource.js]: That's right Adrian, thanks for finding the error!

**Pull request merged**